bugsnag/src/main/java/com/bugsnag/Configuration.java

```diff
@@ -183,4 +189,20 @@ Map<String, String> getSessionApiHeaders() {
         map.put(HEADER_BUGSNAG_SENT_AT, DateUtils.toIso8601(new Date()));
         return map;
     }
+
+    /**
+     * Set the serializer to use for serializing custom objects.
+     * @param serializer the serializer implementation to use
+     */
+    public void setSerializer(ISerializer serializer) {
+        this.serializer = serializer;
+    }
+
+    /**
+     * Get the serializer used for serializing custom objects.
+     * @return the serializer
+     */
+    public ISerializer getSerializer() {
+        return serializer;
+    }
```

Comment:
It seems that simple getting/setting on the Configuration class is done directly using public fields rather than providing methods - the existing setter functions are for more complicated logic, but this only requires assignement.